tests/framework/integration/testing.go

```diff
@@ -84,6 +99,21 @@ func BeforeTest(t testutil.TB, opts ...TestOption) {
 		testutil.RegisterLeakDetection(t)
 	}
 
+	if options.failpoint != nil && len(options.failpoint.name) != 0 {
+		err := gofail.Enable(options.failpoint.name, options.failpoint.payload)
+		switch {
+		case err == gofail.ErrNoExist:
```

Comment:
Is there any other way to distinguish whether gofail is enabled other than checking if enabling failpoint succeeds?
`gofail.ErrNoExist` could be also returned if we made typo in failpoint name.

Reply:
Yeah, there is a way. Once gofail is enabled, the failpoints map in runtime will be registered and `gofail/runtime.List()` will return an non-empty list. Updated in the next rev.